robolectric-resources/src/main/java/org/robolectric/RuntimeEnvironment.java

```diff
@@ -41,4 +43,8 @@ public static String getQualifiers() {
     public static void setQualifiers(String newQualifiers) {
```

Comment:
In occasions api level is not specified, robolectric use api level in sdkConfig
qualifiers += "v" + sdkConfig.getApiLevel();

I think that's why RuntimeEnvironment keep a reference to the qualifier string instead of config.